core/src/main/java/org/stellar/anchor/sep38/Sep38Service.java

```diff
@@ -221,4 +224,133 @@ private String formatAmount(BigDecimal amount, Integer decimals) throws NumberFo
 
     return df.format(newAmount);
   }
+
+  public QuoteResponse postQuote(
+      JwtToken token,
+      String sellAssetName,
```

Comment:
Is there a `Sep38QuoteRequest` class in `dto`?

Reply:
Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/138/commits/0d8d8a239a8483dcda99b0737c0d86f590405134.